src/main/java/org/folio/rest/impl/InitAPIs.java

```diff
@@ -1,27 +1,43 @@
 package org.folio.rest.impl;
 
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.spi.VerticleFactory;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.folio.config.ApplicationConfig;
 import org.folio.dbschema.ObjectMapperTool;
+import org.folio.okapi.common.GenericCompositeFuture;
 import org.folio.rest.resource.interfaces.InitAPI;
 import org.folio.spring.SpringContextUtil;
 
 import com.fasterxml.jackson.databind.DeserializationConfig;
 import com.fasterxml.jackson.databind.SerializationConfig;
 
-import io.vertx.core.AsyncResult;
-import io.vertx.core.Context;
-import io.vertx.core.Future;
-import io.vertx.core.Handler;
-import io.vertx.core.Vertx;
 import io.vertx.core.json.jackson.DatabindCodec;
+import org.folio.verticle.OrderConsumerVerticle;
+import org.folio.verticle.consumers.SpringVerticleFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.support.AbstractApplicationContext;
+
+import java.util.Arrays;
 
 /**
  * The class initializes vertx context adding spring context
  */
 public class InitAPIs implements InitAPI {
-  private final Logger logger = LogManager.getLogger();
+
+  private final Logger LOGGER = LogManager.getLogger();
+
+  private static final String SPRING_CONTEXT_KEY = "springContext";
```

Comment:
why is the spring context key added now, but was not present before?

Reply:
private static final String SPRING_CONTEXT_KEY = "springContext" is private in org.folio.spring.SpringContextUtil (folio-di-support lib).
I need to get a SpringVerticleFactory bean from a Spring context. This operation was not required in the module before. Only in test I found: context.remove("springContext");

Reply:
gotcha.